app/notifications/process_notifications.py

```diff
@@ -118,6 +118,13 @@ def send_notification_to_queue(notification, research_mode, queue=None):
     if research_mode or notification.key_type == KEY_TYPE_TEST:
         queue = QueueNames.RESEARCH_MODE
 
+    if all((
```

Comment:
Could we not just send queue_name=priority from app.service.send_notification.send_one_off_notification?

Reply:
Oh yeah, would be super easy to change this: https://github.com/alphagov/notifications-api/blob/d584ca49498ac28d89460d73115610d856cbb722/app/service/send_notification.py#L78